ui/ducks/send/send.js

```diff
@@ -1661,6 +1687,18 @@ const slice = createSlice({
             addressBook[draftTransaction.recipient.address].name;
         }
       })
+      .addCase(CLEAR_SWAP_AND_SEND_STATE, (state) => {
+        const draftTransaction =
+          state.draftTransactions[state.currentTransactionUUID];
+
+        draftTransaction.quotes = draftTransactionInitialState.quotes;
+        draftTransaction.swapQuotesError =
+          draftTransactionInitialState.swapQuotesError;
+        draftTransaction.isSwapQuoteLoading =
+          draftTransactionInitialState.isSwapQuoteLoading;
+        draftTransaction.swapQuotesLatestRequestTimestamp =
```

Comment:
Can the `latestFetchTime` value be derived from this?

Reply:
`swapQuotesLatestRequestTimestamp1 is the time of the request, not the response. It's used for cleaning up multiple ongoing fetches

Reply:
is there an abort functionality we can use? similar to the AbortController in portfolio

Reply:
I tried that but the quotes kept coming through anyways sometimes IIRC